Week2/homework/maartjes-work.js

```diff
@@ -42,22 +42,18 @@ const tuesday = [
   },
 ];
 
-const maartjesTasks = monday.concat(tuesday);
+const mondayTaskDuration = monday.map(task => task.duration / 60).filter(hour => hour >= 2);
```

Comment:
Although the logic of mapping and filtering tasks is correct, it is supposed to be inside the `computeEarnings` function. Could you please fix it? 🙂